options: add --secondary-sub-ass-override (#13121)
mpv-player/mpv: DOCS/man/options.rst

    This also controls some bitmap subtitle overrides, as well as HTML tags in
    formats like SRT, despite the name of the option.

``--secondary-sub-ass-override=<yes|no|force|scale|strip>``

>     Control whether user secondary substyle overrides should be applied. This
>     is similar to ``--sub-ass-override`` but for secondary subtitles.

Reviewer: ```suggestion
    Control whether user secondary substyle overrides should be applied. This
    works exactly like ``--sub-ass-override``.
```

>     Default: strip.

Reviewer 2: I know that this is just preserving the old behavior but is there any reason to not just default to `yes` now? We already got rid of a hardcoded alignment value for secondary subs so it's technically different anyway.

Author: As mentioned in https://github.com/mpv-player/mpv/pull/13073#issuecomment-1851930198, if we default to retaining the secondary subtitle style, using sub subtitles when there are two ass subtitles and their dialogue subtitles have a pos style will cause them to overlap.

I also have two screenshots here to show the errors encountered in rendering bilingual ass subtitles while retaining the secondary subtitle style.
images: 
![](https://cdn.jsdelivr.net/gh/dyphire/PicGo/img/2023/12/17/20231217024952.png)
![](https://cdn.jsdelivr.net/gh/dyphire/PicGo/img/2023/12/17/20231217025036.png)
I think retaining old behaviors can better handle some special situations.

Reviewer 2: Sure sounds reasonable.
